analysis/estimate/notebooks/compare-estimate-consensus.Rmd

```diff
@@ -250,13 +253,17 @@ ggplot(combined_wide_df) +
 ```
 
 
-```{r, fig.height = 4, fig.width=12}
+```{r, fig.height = 6, fig.width=12}
 ggplot(combined_wide_df) + 
   aes(x = estimate_score_stromal, y = celltype_stromal) + 
   geom_point(aes(shape = sample_quality)) + 
   scale_shape_manual(values = c(19, 21)) +
   geom_smooth(method = "lm") +
-  facet_wrap(vars(project_id), scales = "free_y", nrow = 1) +
+  facet_grid(
+    rows = vars(expression_type),
+    cols = vars(project_id), 
+    scales = "free"
```

Comment:
Hmm the only problem with doing this is that now we are squishing everything for all projects but 17, but maybe that's okay since the values are jus really small. I think probably not worth it to fix at this point.

Reply:
I had the very same sequence of thoughts! Looking again, it's really quick to just make this a patchwork of 2 `facet_wrap()` plots`, so I'll just do that.